Manufacturing Consent: V1.0 from Zip 1014 (#857)
zcash/zips: zip-####.rst

QEDIT has emerged as a global leader in the field of Zero-Knowledge Proofs.
<https://qed-it.com/about-us/>

"ZecHub" refers to the team of content creators who have supported Zcash
through a series of ZCG approved grants.
<https://forum.zcashcommunity.com/t/zechub-an-education-hub-for-zcash-2024-continued/47947>

The terms "Testnet" and "Mainnet" are to be interpreted as described in
section 3.12 of the Zcash Protocol Specification [#protocol-networks]_.

The term "ZSA" is to be interpreted as "Zcash Shielded Assets" the protocol
feature enhancement (and subsequent application layer, legal, maintenance 
efforts, et al) on-going by Qedit, et al.
<https://forum.zcashcommunity.com/t/grant-update-zcash-shielded-assets-monthly-updates/41153>

Abstract
========

This proposal describes a structure for the Zcash Development Fund, to be
enacted in Network Upgrade 6 and last for 4 years. This Dev Fund would consist
of 15% of the block subsidies, as the following allocations:

* 5% for the Zcash Foundation (for internal work and grants);
* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only; 

> and contingent upon an ecosystem mandated "non-direct funding model" for the future;

Reviewer: This is not clear to me - if a non-direct funding model is proposed, how will that proposal interact with this ZIP?

* 4% for Zcash Community Grants for continuation of their current activities as-is;
* 2% for Qedit in support of their ZSA activities, and other Zcash protocol support;

Following the first year, when ECC will no longer receive their 4% allocation, 
those block subsidies will be distributed as 1% to ZCG, 1% to ZecHub, 1% to FPF,
and 1% to ZF.

Governance and accountability are based on existing entities and legal mechanisms,
and increasingly decentralized governance is encouraged.  This proposal mandates 
the ecosystem to design and deploy a "non-direct funding model" as generally
recommended in Josh Swihart's proposal.
<https://forum.zcashcommunity.com/t/zcash-funding-bloc-a-dev-fund-proposal-from-josh-at-ecc/47187>

Upon creation/ activation of a "non-direct funding model" this ZIP should be 

> reconsider by the Zcash ecosystem, to determine if it's ongoing direct block subsidies are 
> a preference. Discussions/ solications/ sentiment gathering from the Zcash

Reviewer: ```suggestion
reconsidered by the Zcash ecosystem, to determine if its ongoing direct block subsidies 
should continuence. Discussions/ solications/ sentiment gathering from the Zcash
```

Author: "continuence" doesn't feel like the correct wording either,  but I agree that the end of that statement can be improved.

"its ongoing direct block subsidies are preferred for continuation."  _(would that be satisfactory?)_

ecosystem should be initiated ~6 months in advance of the presumed
activation of a "non-direct funding model", such that the Zcash ecosystem
preference can be expediently realized.

Block subsidies will be administered through two organizations:
1. Zcash Foundation  (**for ECC, ZCG)
2. Financial Privacy Fund (**for Qedit, ZecHub)

**ZF and FPF adminstration details, costs, et al are currently under debate**
<https://forum.zcashcommunity.com/t/proposal-zcg-under-fpf/48113/11>

Motivation
==========

> Continuing at Zcash's second halving in November 2024, by default 100% of the block

Reviewer: ```suggestion
As of Zcash's second halving in November 2024, by default 100% of the block
```

of independent development teams.

For these reasons, the Zcash Community desires to allocate and
contribute a portion of the block subsidies otherwise allocated to
miners as a donation to support charitable, educational, and
scientific activities within the meaning of Section 501(c)(3).

This proposal also introduces the benefit of a non-USA based entity (FPF) as 

> the administer of block subsidies to two organizations that are also 

Reviewer: ```suggestion
the administrator for block subsidies to two organizations that are also 
```

(e.g., election systems) or entities (for governance or development) for its
execution; but it should strive to support and use these once they are built.

> New Dev Fund recipients should comply with legal, regulatory, and taxation

Reviewer: This should not be restricted to only new dev fund recipients.

Author: Correct, updated.  I had been mistakenly using  "New Dev Fund" to describe the entire concept of a post-halving new dev fund model, but in this case it creates confusion.

Dev Fund allocation
-------------------

Starting at the second Zcash halving in 2024, until the third halving in 2028,
15% of the block subsidy of each block SHALL be allocated to a "Dev Fund" that
consists of the following allocations:

* 5% for the Zcash Foundation (for internal work and grants);
* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only; 
and contingent upon an ecosystem mandated "non-direct funding model" for the future;
* 4% for Zcash Community Grants for continuation of their current activities as-is;
* 2% for Qedit in support of their ZSA activities, and other Zcash protocol support;

Following the first year, when ECC will no longer receive their 4% allocation, 
those block subsidies will be distributed as 1% to ZCG, 1% to ZecHub, 1% to FPF,
and 1% to ZF.

This proposal mandates the ecosystem to design and deploy a "non-direct funding model"
as generally recommended in Josh Swihart's proposal.

> <https://forum.zcashcommunity.com/t/zcash-funding-bloc-a-dev-fund-proposal-from-josh-at-ecc/47187>

Reviewer 2: I'm confused at what is being proposed in this ZIP. It reads like it extends the direct funding model under different allocation & recipients, but then how does this fit with that? Is the idea that a non-direct funding model should be introduced as quickly as possible, and this ZIP would then be amended?

Author: Yes, good points/ questions.  My proposal doesn't anticipate that the creation of a Josh-like non-direct funding mechanism will happen in 1-year,  **but** it could be possible in less than 4-years.  So I'll add some exception handling, to be explicit about the scenario where the non-direct funding model is created in less than 4-years, which should trigger a re-visitation by the ecosystem as to how any further direct funding streams would exist (or not).  In the case of the non-direct funding model being completed, I would suggest that the community decides at that point in time whether or not direct funding should be continued or not.  This proposal recommends 4-years of direct funding, as detailed, regardless of how fast a non-direct funding model can be created/ activated, and would only cause this ZIP to be amended if ecosystem sentiment is a majority for a change to this ZIP's direction.

Dev Fund allocation
-------------------

Starting at the second Zcash halving in 2024, until the third halving in 2028,
15% of the block subsidy of each block SHALL be allocated to a "Dev Fund" that
consists of the following allocations:

* 5% for the Zcash Foundation (for internal work and grants);

> * 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only; 
> and contingent upon an ecosystem mandated "non-direct funding model" for the future;

Reviewer 3: Given that the ZIP mandates the ecosystem to design and deploy a NDFM, the "contingent upon" appears superfluous:
```suggestion
* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only;
```

> and contingent upon an ecosystem mandated "non-direct funding model" for the future;

Reviewer 3: Given that the ZIP mandates the ecosystem to design and deploy a NDFM, the "contingent upon" appears superfluous:
```suggestion
* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only;
```

Author: Thanks,  it seems that we simultaneously caught this one.

* 4% for Zcash Community Grants for continuation of their current activities as-is;
* 2% for Qedit in support of their ZSA activities, and other Zcash protocol support;

Following the first year, when ECC will no longer receive their 4% allocation, 
those block subsidies will be distributed as 1% to ZCG, 1% to ZecHub, 1% to FPF,
and 1% to ZF.

This proposal mandates the ecosystem to design and deploy a "non-direct funding model"
as generally recommended in Josh Swihart's proposal.
<https://forum.zcashcommunity.com/t/zcash-funding-bloc-a-dev-fund-proposal-from-josh-at-ecc/47187>

"Dev Fund" block subsidies will be administered through two organizations:
1. Zcash Foundation  (for ECC, ZCG)
2. Financial Privacy Fund (for Qedit, ZecHub)

The allocations are described in more detail below. The fund flow will be implemented
at the consensus-rule layer, by sending the corresponding ZEC to the designated
address(es) for each block. This Dev Fund will end at the third halving (unless
extended/modified by a future ZIP).

BP allocation (Bootstrap Project)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> These funds SHALL be received and administered by ZF.

Reviewer 3: Is the inclusion of the Bootstrap portion within the ZF's administration something that both Bootstrap and ZF have agreed upon in principle? If yes, this needs a reference to that agreement (given that this is a significant change from ZIP 1014 and ZIP 214). If not, then this would need to be removed (and similarly elsewhere in the ZIP draft).

@daira also makes the point that this increases the risk of key compromise that ZF would need to manage.

Author: At lines 112-117,  I intend to acknowledge the general intention of how direct block subsidies would be administered, and that the details have not been formally agreed upon by parties (presumably) involved (these being ZF, ECC, ZCG, FPF, Qedit, ZecHub).

Given that my ZIP seems to have a minority of support, I don't think it is critical to speculate/ jump into details of how/ who would be administering which block subsidies, or who would receive directly vs. indirectly (hypothetically speaking, if my ZIP gained popularity and were enacted,  the ECC and Qedit would be welcome to change their current opinions, and opt to receive funds directly).

ZF is currently holding the risk of compromise of their and ZCGs funds.  Risk of key compromise is decreases with perpetually devalued ZEC, so by coincidence the ZF and ECC currently have that advantage.  I would certainly entertain the idea of digging into maximizing ZEC administrator opsec if that is (or becomes) a predominant ecosystem concern.

decisions.

> Direct-grant option

Reviewer 3: Just checking that it was intentional to remove the direct-grant option?

Author: Yes, the direct grant option section is obsolete to this ZIP.  It seems obsolete more broadly also, considering the strong community support of NDFM.
